lua/ngx/ssl.lua

```diff
@@ -162,6 +165,23 @@ function _M.server_name()
 end
 
 
+function _M.privkey_pem_to_der(pem)
```

Comment:
For the sake of consistency, I think `priv_key_pem_to_der` is a better name here. We already have a `set_der_priv_key` method name anyway :)